spec/system/form/developer_search_spec.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe 'Developer search form', type: :system do
```

Comment:
This is not wrong.
But actually, you don't need to write this much tests (specs) as a system test.
The system tests check the logic that the unit tests do.
So please think of what you should test in unit tests and system tests.
Because writing tests is not free.
Especially writing system tests take time.

Reply:
@jp-ryuji @thienhv : I has fix it